config/locales/client.en.yml

```diff
@@ -26,8 +21,19 @@ en:
       dismiss: "Dismiss"
       ignore: "Ignore"
       reviewable_delete_prompt: "Are you sure you want to delete this user? This will remove all of their posts and block their email and ip address."
-      topic_deleted: Our automated spam filter, Akismet, has temporarily hidden your topic.
+      topic_deleted: "Our automated spam filter, Akismet, has temporarily hidden your topic."
+
+      user_state:
+        pending: "Akismet hasn't check this user for spam yet."
```

Comment:
```suggestion
        pending: "Akismet hasn't checked this user for spam yet."
```

Reply:
Oops, I just moved these without paying attention to what they said.